### DISCUSSION TITLE
not run on Debian 11 (X86_64)

### DISCUSSION THREAD

**Discussion by jackygoule** [KRTirtho/spotube#1547] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I test to run Spotube on My laptop 
**With faltpack**
fabien@fabien-w55xeu:~$ flatpak run com.github.KRTirtho.Spotube

(process:2): GLib-GObject-CRITICAL **: 12:10:20.083: g_object_new_is_valid_property: object class 'MyApplication' has no property named 'com.github.KRTirtho.Spotube'

** (spotube:2): CRITICAL **: 12:10:20.269: Failed to read XDG desktop portal settings: GDBus.Error:org.freedesktop.portal.Error.NotFound: Requested setting not found

** (spotube:2): CRITICAL **: 12:10:20.272: Failed to read XDG desktop portal settings: GDBus.Error:org.freedesktop.portal.Error.NotFound: Requested setting not found
package:media_kit_libs_linux registered.
flutter: media_kit: WARNING: package:media_kit_native_event_loop not found.
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: FileSystemException: lock failed, path = '/home/fabien/.var/app/com.github.KRTirtho.Spotube/data/spotube/spotube_cache.lock' (OS Error: Ressource temporairement non disponible, errno = 11)
#0      _checkForErrorResponse (dart:io/common.dart:55)
#1      _RandomAccessFile.lock.<anonymous closure> (dart:io/file_impl.dart:1023)
<asynchronous suspension>
#2      StorageBackendVm.initialize (package:hive/src/backend/vm/storage_backend_vm.dart:81)
<asynchronous suspension>
#3      HiveImpl._openBox (package:hive/src/hive_impl.dart:111)
<asynchronous suspension>
#4      HiveImpl.openLazyBox (package:hive/src/hive_impl.dart:160)
<asynchronous suspension>
#5      PersistedStateNotifier.initializeBoxes (package:spotube/utils/persisted_state_notifier.dart:111)
<asynchronous suspension>
#6      main (package:spotube/main.dart:100)
<asynchronous suspension>

[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: FileSystemException: lock failed, path = '/home/fabien/.var/app/com.github.KRTirtho.Spotube/data/spotube/spotube_cache.lock' (OS Error: Ressource temporairement non disponible, errno = 11)
#0      _checkForErrorResponse (dart:io/common.dart:55)
#1      _RandomAccessFile.lock.<anonymous closure> (dart:io/file_impl.dart:1023)
<asynchronous suspension>
#2      StorageBackendVm.initialize (package:hive/src/backend/vm/storage_backend_vm.dart:81)
<asynchronous suspension>
#3      HiveImpl._openBox (package:hive/src/hive_impl.dart:111)
<asynchronous suspension>
#4      HiveImpl.openLazyBox (package:hive/src/hive_impl.dart:160)
<asynchronous suspension>
#5      PersistedStateNotifier.initializeBoxes (package:spotube/utils/persisted_state_notifier.dart:111)
<asynchronous suspension>
#6      main (package:spotube/main.dart:100)
<asynchronous suspension>

With Spotube-linux-x86_64.deb 
fabien@fabien-w55xeu:~/Téléchargements$ sudo apt install ./Spotube-linux-x86_64.deb
[sudo] Mot de passe de fabien : 
Lecture des listes de paquets... Fait
Construction de l'arbre des dépendances... Fait
Lecture des informations d'état... Fait      
Note : sélection de « spotube » au lieu de « ./Spotube-linux-x86_64.deb »
Certains paquets ne peuvent être installés. Ceci peut signifier
que vous avez demandé l'impossible, ou bien, si vous utilisez
la distribution unstable, que certains paquets n'ont pas encore
été créés ou ne sont pas sortis d'Incoming.
L'information suivante devrait vous aider à résoudre la situation : 

Les paquets suivants contiennent des dépendances non satisfaites :
 spotube : Dépend: libjsoncpp25 mais il n'est pas installable
E: Impossible de corriger les problèmes, des paquets défectueux sont en mode « garder en l'état ».

**With nightly**
fabien@fabien-w55xeu:~/Téléchargements/spotube-linux-nightly-x86_64$ ./spotube 
./spotube: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by ./spotube)
./spotube: /lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /home/fabien/Téléchargements/spotube-linux-nightly-x86_64/lib/libflutter_secure_storage_linux_plugin.so)
./spotube: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /home/fabien/Téléchargements/spotube-linux-nightly-x86_64/lib/libflutter_secure_storage_linux_plugin.so)
./spotube: /lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /home/fabien/Téléchargements/spotube-linux-nightly-x86_64/lib/libsystem_tray_plugin.so)
./spotube: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /home/fabien/Téléchargements/spotube-linux-nightly-x86_64/lib/libsystem_tray_plugin.so)
./spotube: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /home/fabien/Téléchargements/spotube-linux-nightly-x86_64/lib/libsystem_tray_plugin.so)

### Expected Behavior

Run Spotube on my laptop

### Steps to reproduce

Test build Spotube on debian 11 x86_64 with flatpack ,.deb and nightly 

### Operating System

debian 11 x86_64

### Spotube version

last

### Installation source

Flathub (Flatpak)

### Additional information

all binary not run

### Self grab

- [X] I'm ready to work on this issue!

**Comment by jackygoule** [comment]
After reboot , spotube (flatpack) run .But when i closed it i can't run another time.Every time i must reboot for run Spotube

**Comment by KRTirtho** [comment]
Spotube is already running in the background (minimized to System Tray). You can disable it from settings.
Close it fully using the system tray icon. If you can't suspend the process from a task manager. Then, reopen Spotube

**Comment by KRTirtho** [comment]
Btw, Minimize System Tray is disabled by default in Linux from 3.6.0

**Comment by jackygoule** [comment]
Thanks for your answer
 it's ok with this setting
